.github/workflows/dev-deploy-worker.yml

```
@@ -0,0 +1,160 @@
+name: Deploy DEV Worker
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - next
+      - main
+    paths:
+      - 'package.json'
+      - 'yarn.lock'
+      - 'apps/api/**'
+      - 'libs/dal/**'
+      - 'libs/shared/**'
+env:
+  TF_WORKSPACE: novu-dev
+
+jobs:
+  test_worker:
+    strategy:
+      matrix:
+        name: ['novu/worker', 'novu/worker-ee']
+    uses: ./.github/workflows/reusable-worker-e2e.yml
+    with:
+      ee: ${{ contains (matrix.name,'ee') }}
+
+  deploy_dev_worker:
+    deploy_dev_webhook:
```

```suggestion
  deploy_dev_worker:
```

Indentation for the whole block needs to be fixed and also this duplicate should be removed